client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -35,7 +35,7 @@ import { recordGoogleEvent } from 'state/analytics/actions';
 const NotificationSubscriptions = createReactClass( {
 	displayName: 'NotificationSubscriptions',
 
-	mixins: [ formBase, observe( 'userSettings' ) ],
+	mixins: [ observe( 'userSettings' ) ],
```

Comment:
We can use this to help test, but I would drop changes to this file before merge.